MetaMask/metamask-extension
# fix: Onboarding: Code style nits

## **Description**

Just doing some code style nits for the onboarding flow

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/27767?quickstart=1)

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. Get to the end of onboarding
2. Try out the advance configuration
3. Ensure all screens toggle as they should.

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
